CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 9.16.2
 
 * Bugfix for Radios component (PR #496)
```

Comment:
I think this also includes the copy to clipboard component, sorry forgot to update the changelog.

Reply:
You're right. Updated to 9.17 and included that PR in the changelog.